test/simulate_test.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2012-2015 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from helpers import unittest
+import luigi
+from luigi.contrib.simulate import RunAnywayTarget
+
+from multiprocessing import Process
+import os
+import tempfile
+
+
+def temp_dir():
+    if os.getenv('TRAVIS') == 'true':
```

Comment:
This feel unintuitive. Can't you do `tempfile.gettempdir()` on Travis? If not. Please say why as a comment.